plugin/user_plugin_server.py

```
@@ -116,6 +118,214 @@ async def plugin_status(plugin_id: Optional[str] = Query(default=None)):
+        # ==========================================
+        # ========== 2. 生命周期事件：startup ==========
+        lifecycle_events = events_by_type.get("lifecycle", {})
+        startup_fn = lifecycle_events.get("startup")
+        if startup_fn:
+            logger.info("Calling lifecycle.startup()")
+            try:
+                if asyncio.iscoroutinefunction(startup_fn):
+                    asyncio.run(startup_fn())
+                else:
+                    startup_fn()
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)